it/mail/chrome/messenger/aboutDialog.dtd

```diff
@@ -4,10 +4,10 @@
 
 <!ENTITY aboutDialog.title          "&brandFullName;">
 
-<!ENTITY productDescription.label    "Realizzato artigianalmente e con amore a San Francisco, California.">
+<!ENTITY productDescription.label    "Realizzato con amore a San Francisco, California.">
```

Comment:
INCOMPLETE. The word "artigianalmente" is not rendered in the new version. Please check the source text.